Architecture/TodoApp/android/mobile/src/main/res/layout/merge_task_detail_view.xml

```diff
@@ -8,10 +8,11 @@
     android:layout_height="match_parent"
     android:orientation="vertical">
 
-    <com.novoda.todoapp.views.TodoAppBar
-      android:id="@+id/app_bar"
+    <com.novoda.todoapp.views.TodoAppBar xmlns:todoApp="http://schemas.android.com/apk/res/com.novoda.todoapp"
+      android:id="@+id/app_bar_with_delete_button"
       android:layout_width="match_parent"
-      android:layout_height="wrap_content" />
+      android:layout_height="wrap_content"
+      todoApp:mergeLayoutId="@layout/merge_app_bar_with_delete_button" />
```

Comment:
Actually @danybony and I figured out you don't need the custom namespace and `res-auto` will let you use custom attributes just fine.